### ISSUE TITLE
Quick question about environment normalization 

### ISSUE THREAD

**Issue by MoMe36** [reinforcement-learning-kr/pg_travel#15]
Hello, 

I'm planning to use your PPO implementations, which seem well-written, clear and easy to understand. But first, I'd like to have the answer to the following question: 

In OpenAI baselines, environments are passed to various classes, such as VecNormalize or Observation/Reward Wrappers or even Monitor. In these cases, observations and rewards are transformed in order to ease learning. However, there is a lot of encapsulation and it makes it kinda difficult to follow the chain. After a quick glance at your implementations, I'm under the impression that you do transform the observations in `unity/utils/running_state.py`. Is that so ? Are there other transformations ? Or were you just careful while designing the environment, designing it to make sure rewards were appropriately scaled ?  

Thanks a lot for your answers.

**Comment by dnddnjs** [comment]
Yes it is. There is no other transformation. The reward function of mujoco and unity walker is already designed well so we didn’t think about scaling reward. But it might affect performance so we will look at it!
